packages/flutter_test/lib/src/controller.dart

```diff
@@ -95,6 +104,104 @@ class SemanticsController {
     return result;
   }
 
+  /// Attemps to find a [SemanticsNode] with the given `action` as close to the
+  /// [SemanticsNode] of `finder` as possible.
+  ///
+  /// The [SemanticsNode] directly related to an [Element] is not necessarily
+  /// the one that can be acted upon. This method finds the directly related
+  /// node and checks whether it supports the given `action`. If the directly
+  /// related node does not support the given `action`, then the closest
+  /// descendent of that node that does support the `action` is returned.
+  ///
+  /// Will throw a [StateError] if:
+  /// * No results are returned from `finder`
+  /// * More than result is returned from `finder`
+  /// * The [SemanticsNode] of `finder` or its descendant nodes do not support
+  ///   `action`
+  ///
+  /// See also:
+  /// * [find]
+  /// * [findWithAnyAction]
```

Comment:
nit: add blank line after "see also" and each bullet should have a quick sentence explaining why one should look there.

Reply:
Updated.